homeassistant/components/gree/const.py

```diff
@@ -0,0 +1,22 @@
+"""Constants for the Gree Climate integration."""
+
+DOMAIN = "gree"
+
+CONNECTION_NETWORK_MAC = "mac"
```

Comment:
This constant can be imported from the device registry helper.
https://github.com/home-assistant/core/blob/a6393fb36c521073bb934932e02a9f75efb43efc/homeassistant/helpers/device_registry.py#L31